### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by borzoni**
Proposed some style and grammar fixes. Nice work, by the way )

### CODE REVIEW

File: en/index.md [hunk 1]
```diff
@@ -333,7 +333,7 @@ I measured size of [DevFest Siberia 2019](https://github.com/TrigenSoftware/DevF
     </tbody>
 </table>
 
-Result is increased loading speed and size of the bundle decreased by ≈20%, [read more detailed report](https://gist.github.com/dangreen/5427c5f2158c357bf0b15d38270508ac). You can also make measurements by yourself — necessary script you can find [in bdsl-webpack-plugin repository](https://github.com/TrigenSoftware/bdsl-webpack-plugin#metrics).
+The result is an increased loading time and bundle size decreased by ≈20%, [read more detailed report](https://gist.github.com/dangreen/5427c5f2158c357bf0b15d38270508ac). You can also make measurements by yourself — you can find required script [in bdsl-webpack-plugin repository](https://github.com/TrigenSoftware/bdsl-webpack-plugin#metrics).
```
Comment (dangreen) [inline en/index.md:336]: increased loading speed ->**decreased** loading time

File: en/index.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ script.src = modernBrowsers.test(navigator.userAgent)
 document.all[1].appendChild(script)
 ```
 
-Another fact that generated regular expressions are faster than matchesUA function from browserslist-useragent, so it makes sense to use browserslist-useragent-regexp on server side too:
+Another fact is that generated regexes are faster than matchesUA function from browserslist-useragent, so it makes sense to use browserslist-useragent-regexp on server side too:
```
Comment (dangreen) [inline en/index.md:145]: regexps ?

Comment (borzoni) [reply en/index.md:145]: изображение

Comment (borzoni) [reply en/index.md:145]: so it doesn't matter, decide on your own taste )

Comment (dangreen) [reply en/index.md:145]: @dangreen "regexp"  it's more JavaScript-way )

File: en/index.md (hunk 1 above)
Comment (borzoni) [reply en/index.md:336]: yeah, i've outwitted myself )

File: en/index.md (hunk 2 above)
Comment (borzoni) [reply en/index.md:145]: I don't remeber for sure why i prefer regex. Mb Java )

**Pull request merged**